docs/release-announcement.md

```diff
@@ -1,11 +1,17 @@
 # Release announcement
 
-## Azure Industrial IoT Platform 2.8 Release
+## Azure Industrial IoT Platform Release 2.8.1
+We are pleased to announce the release of version 2.8.1 of our Industrial IoT Platform components as the first patch update of the 2.8 Long-Term Support (LTS) release. This release contains important security updates, bugfixes and performance optimizations.
 
-We are pleased to announce the release of version 2.8.0 of our Industrial IoT Platform as well as the declaration of Long-Term Support (LTS) for this version.
+> IMPORTANT
+> Please note that OPC Publisher 2.8.1 is not backwards compatible with version 2.5.x. This will be addressed in the next patch release.
+
+## Azure Industrial IoT Platform Release 2.8
+
+We are pleased to announce the release of version 2.8 of our Industrial IoT Platform as well as the declaration of Long-Term Support (LTS) for this version.
```

Comment:
Let's keep `2.8.0` versioning in the body of the announcement for `2.8` as this talks specifically about `2.8.0` release.

Reply:
Didn't we agree that we will use "2.8" for 2.8.0 release. Additionally, convention wise, the number after second decimal denotes the patch version, so it would make sense to use 2.8.1 for first patch version and only 2.8 for LTS release.
@cristipogacean, thoughts?

Reply:
FWIW, I think the way it shows with 2.8 in the bottom, and 2.8.1 above works.

Reply:
@monikavar, I just do not want us to retroactively change our previous release announcements. That announcement was talking about `2.8` LTS release in general, but specifically about `2.8.0` version. So I don't think that we should change what was already there.

Reply:
The change in this part is OK. Resolving this comment.